demo/servers/languages.js

```diff
@@ -25,14 +25,16 @@ let server = http.createServer((req, res) => {
 
 	let html;
 	if(candidates.length === 0) {
-		html = `<p>No results for '${query}'.<p>`;
+		html = `<p data-autocomplete-status="0 search suggestions available">No results for '${query}'.<p>`;
 	} else {
+		let status = candidates.length === 1 ? "1 search suggestion available" : `${candidates.length} search suggestions available`;
```

Comment:
Linting will complain about line length for both of these lines... 🙂 

```suggestion
		html = `
<p data-autocomplete-status="0 search suggestions available">No results for '${query}'.<p>
		`.trim();
	} else {
		let status = candidates.length === 1 ? "1 search suggestion available" :
				`${candidates.length} search suggestions available`;
```

Reply:
updated